tests/integration/ie_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I don't think that `sudo` is needed in this file.
We should not require root access for communication with kubernetes.

Reply:
Done!